sample.env

```
@@ -48,21 +48,19 @@ PG_CLIENT_VERSION=
 # default: 180
 RETRIES_TIMEOUT_MINUTES=
 
-# API key provided in your Airtable personal account used for fetching learning
-# content.
+# Learning content API URL.
 #
 # presence: required
 # type: String
-# default: keyblo10ZCvCqBAJg
-AIRTABLE_API_KEY=
+# default: none
+LCMS_API_URL=
```

Ca, c'est pas terrible: s'il y a un référentiel de test [publié pour l'API](https://github.com/1024pix/pix/blob/dev/api/sample.env), autant l'utiliser
```suggestion
LCMS_API_URL=https://lcms.minimal.pix.fr/api
```